packages/build-deb-packages.sh

```diff
@@ -41,56 +41,60 @@ if [ "X$VERSION" == "X" ]; then
 fi
 FULL_VERSION="${VERSION}-${DEBIAN_VERSION}"
 
-tmpdir=`mktemp -d /tmp/gdr.XXXXXX`
-if [ ! -d "$tmpdir" ]; then
+tmpdir=$(mktemp -d /tmp/gdr.XXXXXX)
+if [ ! -d "${tmpdir}" ]; then
     echo "Failed to create a temp directory!" >&2
     exit 1
 fi
 
 echo "Building gdrcopy debian packages version ${FULL_VERSION} ..."
 
-echo "Working in $tmpdir ..."
+echo "Working in ${tmpdir} ..."
 
 ex cd ${TOP_DIR_PATH}
 
-ex mkdir -p $tmpdir/gdrcopy
-ex rm -rf $tmpdir/gdrcopy/*
-ex cp -r Makefile README.md include src tests LICENSE config_arch packages/debian $tmpdir/gdrcopy/
-ex cp README.md $tmpdir/gdrcopy/debian/README.Debian
-ex cp README.md $tmpdir/gdrcopy/debian/README.source
-ex rm -f $tmpdir/gdrcopy_${VERSION}.orig.tar.gz
+ex mkdir -p ${tmpdir}/gdrcopy
+ex rm -rf ${tmpdir}/gdrcopy/*
+ex cp -r Makefile README.md include src tests LICENSE config_arch packages/debian ${tmpdir}/gdrcopy/
+ex cp README.md ${tmpdir}/gdrcopy/debian/README.Debian
+ex cp README.md ${tmpdir}/gdrcopy/debian/README.source
+ex rm -f ${tmpdir}/gdrcopy_${VERSION}.orig.tar.gz
 
-ex cd $tmpdir/gdrcopy
+ex cd ${tmpdir}/gdrcopy
 ex find . -type f -exec sed -i "s/@VERSION@/${FULL_VERSION}/g" {} +
 
-ex cd $tmpdir
+ex cd ${tmpdir}
 ex mv gdrcopy gdrcopy-${VERSION}
 ex tar czvf gdrcopy_${VERSION}.orig.tar.gz gdrcopy-${VERSION}
 
-ex cd $tmpdir/gdrcopy-${VERSION}
+ex cd ${tmpdir}/gdrcopy-${VERSION}
 ex debuild --set-envvar=CUDA=${CUDA} --set-envvar=PKG_CONFIG_PATH=${PKG_CONFIG_PATH} -us -uc
 
 echo
 echo "Building dkms module ..."
-ex cd $tmpdir/gdrcopy-${VERSION}/src/gdrdrv
+ex cd ${tmpdir}/gdrcopy-${VERSION}/src/gdrdrv
 ex make clean
 
-ex mkdir -p $tmpdir/gdrdrv-dkms-${VERSION}/
-ex cp -r $tmpdir/gdrcopy-${VERSION}/src/gdrdrv $tmpdir/gdrdrv-dkms-${VERSION}/gdrdrv-${VERSION}
-ex cp ${SCRIPT_DIR_PATH}/dkms.conf $tmpdir/gdrdrv-dkms-${VERSION}/gdrdrv-${VERSION}/
```

Comment:
I argue that the driver source code must not change when simply changing DEBIAN_VERSION, hence the dir should only versioned with MAJOR.MINOR